packages/runtime/test/transport/relationships.test.ts

```diff
@@ -281,3 +291,83 @@ describe("Attributes for the relationship", () => {
         );
     });
 });
+
+describe("RelationshipTermination", () => {
+    let relationshipId: string;
+    let terminationResult: Result<RelationshipDTO, ApplicationError>;
+    beforeAll(async () => {
+        relationshipId = (await ensureActiveRelationship(services1.transport, services2.transport)).id;
+
+        const requestContent = {
+            content: {
+                items: [
+                    {
+                        "@type": "TestRequestItem",
+                        mustBeAccepted: false
+                    }
+                ]
+            },
+            peer: services2.address
+        };
+        await exchangeMessageWithRequest(services1, services2, requestContent);
+
+        terminationResult = await services1.transport.relationships.terminateRelationship({ relationshipId });
+    });
+
+    test("relationship status is terminated", async () => {
+        expect(terminationResult).toBeSuccessful();
+        const result = (await services1.transport.relationships.getRelationship({ id: relationshipId })).value;
+        expect(result.status).toBe(RelationshipStatus.Terminated);
+    });
+
+    test("should not send a message", async () => {
+        const result = await services1.transport.messages.sendMessage({
+            recipients: [services2.address],
+            content: {
+                "@type": "Mail",
+                body: "b",
+                cc: [],
+                subject: "a",
+                to: [services2.address]
+            }
+        });
+        expect(result).toBeAnError(/.*/, "error.transport.messages.noMatchingRelationship");
+    });
+
+    test("should not decide a request", async () => {
+        await syncUntilHasRelationships(services2.transport);
+        const incomingRequest = (await services2.eventBus.waitForEvent(IncomingRequestReceivedEvent)).data;
+
+        const canAcceptResult = (await services2.consumption.incomingRequests.canAccept({ requestId: incomingRequest.id, items: [{ accept: true }] })).value;
+        expect(canAcceptResult.isSuccess).toBe(false);
+        expect(canAcceptResult.code).toBe("error.consumption.requests.noMatchingRelationship");
```

Comment:
please use the utility method toBeAnError instead of doing everything manually. Same for the other occurences.

Reply:
the `canAccept/Reject` return validation results though

Reply:
you can make the function also accept these.